src/input/source/iio/bmi_imu.rs

```diff
@@ -84,13 +94,14 @@ impl IMU {
                     }
 
                     // Sleep between each poll iteration
-                    let duration = time::Duration::from_micros(250);
+                    let duration = time::Duration::from_millis(40);
```

Comment:
Do we want to change this back to 250µs? In the future, we can get this from the driver interface based on polling frequency.